test/color/test_hls.py

```diff
@@ -18,15 +16,40 @@ class TestRgbToHls:
 
     def test_rgb_to_hls(self):
 
-        data = torch.rand(3, 5, 5)
+        data = torch.tensor([[[0.4237059, 0.1935902, 0.8585021, 0.3790484, 0.1389151],
+                              [0.5933651, 0.0474544, 0.2801555, 0.1691061, 0.9221829],
+                              [0.2351739, 0.5852075, 0.5789326, 0.8411915, 0.5960411],
+                              [0.0290176, 0.6459382, 0.8581501, 0.4755400, 0.7735767],
+                              [0.9497226, 0.0919441, 0.5462211, 0.7836787, 0.6403612]],
```

Comment:
after recent PR you to:
```python
def test_rgb_to_hls(self, device):
   data = ().to(device)
```

and make sure the tests pass in gpu, from root: `make test-gpu`